bin/curriculum/export/README.md

```diff
@@ -36,63 +36,69 @@ To export progress for a given unit:
 1. check the [unit progress shares](https://docs.google.com/spreadsheets/d/1BiK3a3rlEEto1x9_ITjX7l0CsbhO0WZQrdYNdISWKQA/edit)
    gsheet to confirm that the data you need is not already available.
 
-2. connect to production-daemon
+2. choose a descriptive name for your new dataset indicating the unit name and date range:
+  - `<unit-name>-ending-YYYY-MM-DD` (e.g. `csd3-2024-ending-2025-04-01` for units less than 1 year old)
+  - `<unit-name>-school-year-2023` (prefer capturing one entire school year at a time for units > 1 year old)
+  -  see [unit progress shares](https://docs.google.com/spreadsheets/d/1BiK3a3rlEEto1x9_ITjX7l0CsbhO0WZQrdYNdISWKQA/edit)
+     gsheet for more details and examples.
+
+3. connect to production-daemon
 ```bash
 ssh -t gateway ssh production-daemon
 cd production
 ```
 
-3. export unit progress from redshift
+4. export unit progress from redshift
 ```bash
-SKIP_SCRIPT_PRELOAD=1 bin/curriculum/export/export_unit_progress.rb -u unit_name
+SKIP_SCRIPT_PRELOAD=1 bin/curriculum/export/export_unit_progress.rb -u unit-name -o unit-name-date-range
```

Comment:
Super non-critical, but fyi I was unsure here and wouldn't mind having an example. From the above its not clear to me if `unit-name-date-range=<unit-name>-ending-YYYY-MM-DD` or `unit-name-date-range=ending-YYYY-MM-DD`. I /think/ its just a human given name and arbitrary, but I'm also not 100% sure of that.

I wouldn't mind using `<unit-name-date-range>` as the placeholder, and having a very clear definition in 2 of what this variable should be.

I also wouldn't mind if it was committed as-is, I'm sure I could figure this out if I had to run the script, and this hardly needs to be A+ grade docs.

Reply:
addressed in a249f3c9859294185d88bb0af95e402f264a9d39 and e8d2a7738ac604a33a614000596c6c4dc87e61e8

Reply:
sorry those commits have been orphaned. addressed in https://github.com/code-dot-org/code-dot-org/pull/65538/commits